### ISSUE TITLE
1.19.2 FORGE: Game crashes when used with Cloth Config API

### ISSUE THREAD

**Issue by ziolkoplays** [Virtuoel/Pehkui#396]
### Minecraft version(s)

1.19.2

### Minecraft version details

1.19.2

### Mod loader

Forge

### Mod loader version

Forge 43.2.0

### In what kind of world or server did the problem occur?

Other (specify in "What went wrong?" section)

### What went wrong? (Crash logs don't go here)

I pressed the Play button on the launcher hoping to use the immersive portals (For forge) which requires Cloth Config; but instead got this crash report: Error: java.lang.NoClassDefFoundError: Could not initialize class net.minecraft.client.renderer.RenderType Exit code: -1

### Full list of installed mods and their version numbers

Cloth Config API: 8.2.88
Create: 0.5.0
Create Cafe: 1.1.5
Create Crafts & Additions: Not specified.
Create Deco: 1.2.12
Create Stuff & Additions: 2.0.2
Immersive Portals (For forge:) 2.3.3
Pehkui: 3.7.2
Size Devices: 1.0.2
WorldEdit: 7.2.12

### Did the problem cause the game to crash?

Yes. (paste crash report below)

### Full crash report contents (if you experienced a crash)

<details>
<summary>Crash report output:</summary>
<p>
The game crashed whilst initializing game
Error: java.lang.NoClassDefFoundError: Could not initialize class net.minecraft.client.renderer.RenderType
[crash-2023-04-03_16.44.51-client.txt](https://github.com/Virtuoel/Pehkui/files/11142781/crash-2023-04-03_16.44.51-client.txt)
Exit code: -1

**Comment by Mysticpasta1** [comment]
crash report: [crash-2023-04-04_08.15.45-client.txt](https://github.com/Virtuoel/Pehkui/files/11148956/crash-2023-04-04_08.15.45-client.txt)

same crash but it is a mixin crash from pehkui causing the RenderType not to load, this only occurs when Immersive Portals is installed

mixin crash :
```
[04Apr2023 08:15:41.283] [modloading-worker-0/FATAL] [mixin/]: Mixin apply failed pehkui.mixins.json:client.compat115plus.compat1192minus.GameRendererMixin -> net.minecraft.client.renderer.GameRenderer: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Specified index 0 for @ModifyArg is invalid for args (Lcom/mojang/blaze3d/vertex/PoseStack;DDD), expected D on net/minecraft/client/renderer/GameRenderer::pehkui$bobView$translate$x [INJECT Applicator Phase -> pehkui.mixins.json:client.compat115plus.compat1192minus.GameRendererMixin -> Apply Injections ->  -> Inject -> pehkui.mixins.json:client.compat115plus.compat1192minus.GameRendererMixin->@ModifyArg::pehkui$bobView$translate$x(D)D]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Specified index 0 for @ModifyArg is invalid for args (Lcom/mojang/blaze3d/vertex/PoseStack;DDD), expected D on net/minecraft/client/renderer/GameRenderer::pehkui$bobView$translate$x [INJECT Applicator Phase -> pehkui.mixins.json:client.compat115plus.compat1192minus.GameRendererMixin -> Apply Injections ->  -> Inject -> pehkui.mixins.json:client.compat115plus.compat1192minus.GameRendererMixin->@ModifyArg::pehkui$bobView$translate$x(D)D]
	at org.spongepowered.asm.mixin.injection.invoke.ModifyArgInjector.findArgIndex(ModifyArgInjector.java:158) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
```

latest.log : 
[2023-04-04-1.log](https://github.com/Virtuoel/Pehkui/files/11148994/2023-04-04-1.log)

**Comment by Mysticpasta1** [comment]
downgrading to 3.7.0-1.19.2 forge fixes it, had to be either 3.7.1 or 3.7.2 changes that breaks it

**Comment by Virtuoel** [comment]
Shoud now be resolved as of the 3.7.3 release.
